### ISSUE TITLE
Alerting based on number of resque workers

### ISSUE THREAD

**Issue by evelant** [railsware/newrelic_platform_plugins#20]
Setting an alert in newrelic based on the number of background workers alerts when the number of background workers is higher than the set value rather than lower. Shouldn't the alert work the other way around? If background worker count drops below the set value it should alert rather than alert when background worker count is above the set value.

**Comment by ozsynergy** [comment]
Yes! It would be really good if this could be fixed - still an issue in 2015.

**Comment by dingn1** [comment]
Yes! It is still the case. We need to have an alert when worker is down.
